### PR TITLE
removed all c# microservice SDK references

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/microservice-sdk/microservices-best-practices.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following list is a collection of best practices you should take into consid
 
 #### Microservice SDK {#microservice-sdk}
 
-Whenever possible, use the {{< product-c8y-iot >}} Microservice SDK as it builds a lot of functionality. It is fully open source and can be extended as required. The {{< product-c8y-iot >}} Microservice SDK can be found [here](https://github.com/SoftwareAG/cumulocity-clients-java/tree/develop/microservice). See the relevant chapters of this guide for further instructions on the [Microservice SDK for Java](/microservice-sdk/java/) and the [Microservice SDK for C#](/microservice-sdk/cs/).
+Whenever possible, use the {{< product-c8y-iot >}} Microservice SDK as it builds a lot of functionality. It is fully open source and can be extended as required. The {{< product-c8y-iot >}} Microservice SDK can be found [here](https://github.com/SoftwareAG/cumulocity-clients-java/tree/develop/microservice). See also the relevant chapters of this guide for further instructions on the [Microservice SDK for Java](/microservice-sdk/java/).
```
Comment (BeateRixen) [inline content/microservice-sdk/microservices-best-practices.md:14]:
```suggestion
Whenever possible, use the {{< product-c8y-iot >}} Microservice SDK as it builds a lot of functionality. It is fully open source and can be extended as required. The {{< product-c8y-iot >}} Microservice SDK can be found [here](https://github.com/SoftwareAG/cumulocity-clients-java/tree/develop/microservice). See also [Microservice SDK for Java](/microservice-sdk/java/) for further instructions.
```

File: content/microservice-sdk/http-bundle/node-js-microservice.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Node.js microservice
 layout: redirect
 ---
 
-{{< product-c8y-iot >}} provides SDKs for developing microservices using C# or Java. Nevertheless, you are free to choose the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/general-aspects/#requirements).
+{{< product-c8y-iot >}} provides a SDK for developing microservices using Java. Nevertheless, you are free to choose the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/general-aspects/#requirements).
```
Comment (BeateRixen) [inline content/microservice-sdk/http-bundle/node-js-microservice.md:7]:
```suggestion
{{< product-c8y-iot >}} provides an SDK for developing microservices using Java. Nevertheless, you are free to choose the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/general-aspects/#requirements).
```

File: content/microservice-sdk/rest-bundle/rest-overview.md [hunk 3]
```diff
@@ -12,14 +12,15 @@ REST is a very simple and secure protocol based on HTTP(S) and TCP. It is today
...
 
 - [REST implementation](https://{{< domain-c8y >}}/api/core/#section/REST-implementation) is the reference for all general concepts.
 - [Fragment library](/device-integration/fragment-library/) specifies the data model for device management.
 - [Sensor library](/concepts/domain-model/#sensor-library) specifies the data model for sensors and controls.
 
-If you develop applications using Java or C#, check these relevant sections for even more convenient access to {{< product-c8y-iot >}}'s functionality.
-For further information on REST interfaces in general and on integrating devices with REST, refer to the [REST section](/device-integration/rest).
+If you develop applications using Java, check [Microservice SDK for Java](/microservice-sdk/java/#overview) for even more convenient access to {{< product-c8y-iot >}}'s functionality.
+
+For further information integrating devices using REST, refer to [Device integration using REST](/device-integration/rest).
```
Comment (BeateRixen) [inline content/microservice-sdk/rest-bundle/rest-overview.md:23]:
```suggestion
For further information on integrating devices using REST, refer to [Device integration using REST](/device-integration/rest).
```

File: content/concepts/applications-bundle/microservices.md [hunk 4]
```diff
@@ -19,8 +19,4 @@ The hosting of the microservice is provided by {{< product-c8y-iot >}}. This way
...
-
-{{< c8y-admon-info >}}
-For Microservice developers {{< company-c8y >}} provides Microservice SDK in [Java programming language](/microservice-sdk/java) and [C# programming language](/microservice-sdk/cs) for .Net Core. Refer to the relevant sections in [Microservice SDK](/microservice-sdk/microservice-sdk-introduction/).
-{{< /c8y-admon-info >}}
+For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to [Microservice SDK](/microservice-sdk/microservice-sdk-introduction/) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For Microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java) for .Net Core.
```
Comment (BeateRixen) [inline content/concepts/applications-bundle/microservices.md:22]:
```suggestion
For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to [Microservice SDK](/microservice-sdk/microservice-sdk-introduction/) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For Microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java).
```

**Review by rssag — APPROVED** [review]
1 suggestion + approval

**Review by MWindel — APPROVED** [review]

**Pull request merged**